app/views/users/index.html.erb

```diff
@@ -1 +1,13 @@
-<h1>Users</h1>
\ No newline at end of file
+<h1>Users</h1>
+<ul>
+  <% @users.each do |user| %>
+  <li>
+    <a href="<%= user_posts_path(user) %>">
+      <div class="user-box">
+        <%= render partial: 'shared/user_photo', locals: { user: user } %>
+        <%= render partial: 'shared/user_details', locals: { user: user } %>
+     </div>
+  </a>
+  </li>
+  <% end %>
```

Comment:
- [ ] Based on the requirements, when I click on a user on the `Users#Index` page, it should lead me to the `Users#Show` page but your application leads to the `Posts#Index` page. Kindly correct this. 👍 😎